README.md

```diff
@@ -47,4 +47,31 @@ Then, the project can be built with:
 
 ```bash
 make
-```
\ No newline at end of file
+```
+
+## How to deploy
+
+### Prerequisites
+
+Before deploying Merak, you will need the following.
+
+- A Kubernetes cluster with [flannel](https://github.com/flannel-io/flannel) installed
+- [Linkerd](https://linkerd.io/2.12/getting-started/) installed on the cluster (Needed for [Alcor](https://github.com/futurewei-cloud/alcor))
+
+
+### Deployment
+
+Once your cluster is ready, you can deploy the latest small scale development (one replica for every service) build of Merak and Alcor with the command below.
+
+```
+kubectl kustomize https://github.com/futurewei-cloud/merak/deployments/kubernetes/dev --enable-helm | kubectl apply -f -
+```
+
+A successful deployment should take roughly 5 minutes for all pods to come to `running` state.
+
```

Comment:
@phudtran  please add one more line before line70 to explain what components will be deployed in this script.
This script will deploy meshnet-cni daemonset, alcor microservices and their ignite-db instances, and Merak microservices and their redis db instances.

Reply:
Ok I've added this information